docs/guide/rest-resources.md

```diff
@@ -244,4 +244,64 @@ will also include the pagination information by the following HTTP headers:
 * `X-Pagination-Per-Page`: The number of resources in each page;
 * `Link`: A set of navigational links allowing client to traverse the resources page by page.
 
+Since collection in REST APIs is a data provider, it shares all data provider features i.e. pagination and sorting.
+
 An example may be found in the [Quick Start](rest-quick-start.md#trying-it-out) section.
+
+### Filtering collections <span id="filtering-collections"></span>
+
+Since version 2.0.13 Yii provides a facility to filter collections. In case you are using `ActiveController` you can enjoy it
+right away. An example can be found in the [Quick Start](rest-quick-start.md#trying-it-out) section. In case you're
+implementing an endpoint yourself, filtering could be done like the following:
+
+```php
+$filter = new ActiveDataFilter([
```

Comment:
We need tell more about `DataFilter::attributeMap`. 

This property `attributeMap` (not `searchModel`) can be use (additional) to resolve alias, who we use in resources definition
```php
                    'attributeMap' => [
                        'aliasId' => 'realFieldId',
                    ],
```
```php
class \api\Post extends \common\Post {
    public function fields(){
        $fields = parent::fields();
        unset($fields['realFieldId']);
        $fields['aliasId'] = function () {
            return $this->realFieldId;
        };
        return $fields;
    }
}
```

Reply:
So... I think we need move $attributeMap to from controller (definition of DataFilter) to the model `searchModel`
```php
class \api\models\PostSearch extends app\models\Post {
    public static function attributeMap(){
         return ['aliasId' => 'realFieldId'];
    }
    public function fields(){
        $fields = parent::fields();
        unset($fields['realFieldId']);
        $fields['aliasId'] = function () {
            return $this->realFieldId;
        };
        return $fields;
    }
}
```
```php
$filter = new ActiveDataFilter([ 
'searchModel' => \api\models\PostSearch::class,
'attributeMap' => [ \api\models\PostSearch::class, 'attributeMap'],
]);
```

i.e. we need closure support of `attributeMap` property too